### DISCUSSION TITLE
Control characters support?

### DISCUSSION THREAD

**Discussion by shorton3** [carlmontanari/scrapli#207] [Q&A]
Hello, I'm pretty new to looking at Scrapli; thanks for the contribution.  Does Scrapli support being able to send control-characters through the connection to the remote device?  By control characters, I mean non-printable characters such as Ctrl+], Ctrl+Alt+Del, Ctrl+C, and such.  Thanks!

**Comment by carlmontanari** [comment]
hey @shorton3 scrapli doesn't do anything to the inputs you provide -- we just write whatever you give us basically! So, "yes" should be able to do that no problemo!
